openpype/hosts/maya/plugins/publish/extract_multiverse_usd_over.py

```diff
@@ -0,0 +1,142 @@
+import os
+
+import avalon.maya
+import openpype.api
+
+from maya import cmds
+
+
+class ExtractMultiverseUsdOverride(openpype.api.Extractor):
+    """Extractor for USD Override by Multiverse."""
+
+    label = "Extract Multiverse USD Override"
+    hosts = ["maya"]
+    families = ["usdOverride"]
+
+    @property
+    def options(self):
+        """Overridable options for Multiverse USD Export
+
+        Given in the following format
+            - {NAME: EXPECTED TYPE}
+
+        If the overridden option's type does not match,
+        the option is not included and a warning is logged.
+
+        """
+
+        return {
+            "writeAll": bool,
+            "writeTransforms": bool,
+            "writeVisibility": bool,
+            "writeAttributes": bool,
+            "writeMaterials": bool,
+            "writeVariants": bool,
+            "writeVariantsDefinition": bool,
+            "writeActiveState": bool,
+            "writeNamespaces": bool,
+            "writeTimeRange": bool,
+            "timeRangeStart": int,
+            "timeRangeEnd": int,
+            "timeRangeIncrement": int,
+            "timeRangeNumTimeSamples": int,
+            "timeRangeSamplesSpan": float,
+            "timeRangeFramesPerSecond": float
+        }
+
+    @property
+    def default_options(self):
+        """The default options for Multiverse USD extraction."""
+        start_frame = int(cmds.playbackOptions(query=True,
+                                               animationStartTime=True))
+        end_frame = int(cmds.playbackOptions(query=True,
+                                             animationEndTime=True))
+
+        return {
+            "writeAll": False,
+            "writeTransforms": True,
+            "writeVisibility": True,
+            "writeAttributes": True,
+            "writeMaterials": True,
+            "writeVariants": True,
+            "writeVariantsDefinition": True,
+            "writeActiveState": True,
+            "writeNamespaces": False,
+            "writeTimeRange": True,
+            "timeRangeStart": start_frame,
+            "timeRangeEnd": end_frame,
+            "timeRangeIncrement": 1,
+            "timeRangeNumTimeSamples": 0,
+            "timeRangeSamplesSpan": 0.0,
+            "timeRangeFramesPerSecond": 24.0
+        }
+
+    def process(self, instance):
+        # Load plugin firstly
+        cmds.loadPlugin("MultiverseForMaya", quiet=True)
+
+        # Define output file path
+        staging_dir = self.staging_dir(instance)
+        file_name = "{}.usda".format(instance.name)
+        file_path = os.path.join(staging_dir, file_name)
+        file_path = file_path.replace('\\', '/')
+
+        # Parse export options
+        options = self.default_options
+        self.log.info("Export options: {0}".format(options))
```

Comment:
As stated above in another comment you are currently using only the default options and are ignoring all settings on the created instance. I believe you're missing [something like this logic](https://github.com/pypeclub/OpenPype/blob/develop/openpype/hosts/maya/plugins/publish/extract_fbx.py#L117-L143) to be used.